modules/geo/src/main/java/org/opensearch/geo/GeoModulePlugin.java

```diff
@@ -105,23 +102,4 @@ public List<CompositeAggregationSpec> getCompositeAggregations() {
             )
         );
     }
-
-    /**
-     * Registering the GeoBounds Aggregation on the GeoShape Field. This function allows plugins to register new
-     * aggregations using aggregation names that are already defined in Core, as long as the new aggregations target
-     * different ValuesSourceTypes.
-     *
-     * @return A list of the new registrar functions
-     */
-    @Override
-    public List<Consumer<ValuesSourceRegistry.Builder>> getAggregationExtentions() {
```

Comment:
Could you please clarify why this registration has been removed? I do not 100% understand the consequences yet, but the comment documents the intention pretty well why it is needed: `GEO_SHAPE` is defined in core.

Reply:
It was removed because I move it to [GeoBoundsAggregatorFactory.java](https://github.com/opensearch-project/OpenSearch/pull/5589/files#diff-8fa6a1ff75628d10d0b4dacc71c37f8abed5daed9cdcc7f313b556cdbe070ff2). When I was working on GeoBounds Aggregation I was thinking that to add new aggregations I need to put it in getAggregationExtentions(). But while I am working on GeoTile and GeoHash I realized that these aggregations can be added in *AggregatorFactory.class . The PR contains that change. 

I hope this clarifies.

Reply:
Thanks, yeah, I saw that, really unsure about the consequences but the change (`AggregatorFactory`) looks good

Reply:
I tested the changes via IT and also manual testing it works. Also, AggregatorFactory is used to register the aggregation. Given that this is same aggregation but on a different SourceValue type I don't see any issue. The only thing I can think of is, is this the right place from a code abstraction standpoint. I will wait for @nknize to comment on this.

Reply:
But as per my understanding of the Aggregation framework, I think this looks good, as aggregations are present in same module/plugin. So we are good. getAggregationExtentions : function is provided to extend the aggregations by the plugin for a different Source value type but having same aggregation name defined in core. Doc link: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/plugins/SearchPlugin.java#L174-L176